diff --git a/third-party/SDL/src/video/uikit/SDL_uikitview.m b/third-party/SDL/src/video/uikit/SDL_uikitview.m
index 75c01b8..74e27d4 100644
--- a/third-party/SDL/src/video/uikit/SDL_uikitview.m
+++ b/third-party/SDL/src/video/uikit/SDL_uikitview.m
@@ -179,6 +179,12 @@ extern int SDL_AppleTVRemoteOpenedAsJoystick;
         return [UIPointerStyle hiddenPointerStyle];
     }
 }
+
+- (void)indirectPointerMoving:(UITouch *)touch API_AVAILABLE(ios(13.4))
+{
+    CGPoint point = [self touchLocation:touch shouldNormalize:NO];
+    SDL_SendMouseMotion(sdlwindow, 0, 0, (int)point.x, (int)point.y);
+}
 #endif /* !TARGET_OS_TV && __IPHONE_13_4 */
 
 - (SDL_TouchDeviceType)touchTypeForTouch:(UITouch *)touch
@@ -240,6 +246,7 @@ extern int SDL_AppleTVRemoteOpenedAsJoystick;
                 if (!SDL_HasGCMouse()) {
                     int i;
 
+                    [self indirectPointerMoving:touch];
                     for (i = 1; i <= MAX_MOUSE_BUTTONS; ++i) {
                         if ((event.buttonMask & SDL_BUTTON(i)) != 0) {
                             Uint8 button;
@@ -295,6 +302,7 @@ extern int SDL_AppleTVRemoteOpenedAsJoystick;
                 if (!SDL_HasGCMouse()) {
                     int i;
 
+                    [self indirectPointerMoving:touch];
                     for (i = 1; i <= MAX_MOUSE_BUTTONS; ++i) {
                         if ((event.buttonMask & SDL_BUTTON(i)) != 0) {
                             Uint8 button;
@@ -362,7 +370,7 @@ extern int SDL_AppleTVRemoteOpenedAsJoystick;
 #if !TARGET_OS_TV && defined(__IPHONE_13_4)
         if (@available(iOS 13.4, *)) {
             if (touch.type == UITouchTypeIndirectPointer) {
-                /* Already handled in pointerInteraction callback */
+                [self indirectPointerMoving:touch];
                 handled = YES;
             }
         }
